libraries/stumbler/src/main/java/org/mozilla/mozstumbler/service/stumblerthread/scanners/ScanManager.java

```diff
@@ -53,34 +52,41 @@ public void batteryCheckCallback(BatteryCheckReceiver receiver) {
     };
     private LocationChangeSensor mLocationChangeSensor;
     private MotionSensor mMotionSensor;
-    private boolean mIsMotionlessPausedState;
+
+    private enum ScannerState {
```

Comment:
The Scanner has been finicky before with buggy hardware, so I'd like to be able to inspect the value of `mScannerState` when we put the ScanManager under test.

Can we make `ScannerState` and `mScannerState` to have package visibility?